src/app/components/fulltime/FullTime.tsx

```
@@ -278,12 +290,15 @@ const FullTime = () => {
       if (single.jungang.length !== 0) {
         single.count++
       }
+      if (single.directY.length !== 0) {
+        single.count++
+      }
 
       filteredByType.push(single)
       return <></>
       // [{ time: '08', direct: ["08:00", "08:10", ...], circle: [], directY: ["08:20", "08:50"] }, { time: '09', direct: [], circle: [], directY: [] }, ...]
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)